changes/add_GLT-4046

```diff
@@ -0,0 +1 @@
+Sort by QC status in all QC step tables
```

Comment:
The sort was not added to all QC step tables - it's not on analysis review, release approval, or release. Please update the change message to specify receipt, extraction, library qualification, and full-depth sequencing tables.